source/_posts/2015-06-03-technical-thursdays.markdown

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: Technical Thursdays
```

Comment:
@spastorino I don't like that title that much, because this post does not cover all the "continuous learning practices at WyeWorks".
Plus, `Technical Thursday` is catchier :)